src/collections/blog/Blog.style.js

```diff
@@ -1,207 +1,206 @@
 import styled from "styled-components";
```

Comment:
@blossom-babs were any changes done to this file? If so, could you highlight the line(s) for review? If not, perhaps, you might remove this file from the PR (revert the change on this file). 

Thank you for getting this posted, Blossom. 💯

Reply:
The change here that she did was adding `line-spacing` to the `p` elements, which shouldn't be done, here's the [commit](https://github.com/layer5io/layer5/pull/1806/commits/a0accad71ce04b2602f509971c6dd5b91426fafe). And instead, the MDX should be updated.

Reply:
@Nikhil-Ladha I added line spacing to the hero,  do you suggest I remove the style?

Reply:
@blossom-babs you just need to add a space between 'service mesh performance' & 'and' and not the entire hero.